unyt/dask_array.py

```diff
@@ -8,20 +8,13 @@
 from functools import wraps
 
 import numpy as np
+import pytest
```

Comment:
this makes `pytest` a hard dependency for anyone attempting to use `unyt.dask_array`, doesn't it?

Reply:
Good call, I actually missed that. I made some attempts to use protocols defined in pytest-doctest-plus but this is the only solution I found. I'll try to refine it.